pkg/common/path.go

```diff
@@ -137,28 +137,31 @@ func FilterInputFile(in io.Reader, tmpDir string) error {
 }
 
 // expandDir takes a single package directory as a parameter, and returns
-// an array of config file paths excluding the inventory object. Returns
-// an error if one occurred while processing the paths.
-func expandDir(dir string) ([]string, error) {
+// the inventory template filepath and an array of config file paths. If no
+// inventory template file, then the first return value is an empty string.
+// Returns an error if one occurred while processing the paths.
+func ExpandDir(dir string) (string, []string, error) {
 	filepaths := []string{}
 	r := kio.LocalPackageReader{PackagePath: dir}
 	nodes, err := r.Read()
 	if err != nil {
-		return filepaths, err
+		return "", filepaths, err
 	}
+	var invFilepath string
 	for _, node := range nodes {
 		meta, err := node.GetMeta()
 		if err != nil {
 			continue
 		}
+		path := meta.Annotations[kioutil.PathAnnotation]
+		path = filepath.Join(dir, path)
```

Comment:
Does this work for packages with subfolders? It might be worth adding a test for this.

Reply:
Added unit test for this case.